runestone/activecode/activecode.py

```diff
@@ -126,7 +127,8 @@ class ActiveCode(Directive):
         'timelimit': directives.unchanged,
         'stdin' : directives.unchanged,
         'datafile' : directives.unchanged,
-        'sourcefile' : directives.unchanged
+        'sourcefile' : directives.unchanged,
+        'available_files' : directives.unchanged
```

Comment:
Is this a list of div_ids for datafiles that have been saved or what should the value of this be?